cryptocurrencychallenge/src/main/java/com/yoti/android/cryptocurrencychallenge/market/MarketFragment.kt

```diff
@@ -0,0 +1,18 @@
+package com.yoti.android.cryptocurrencychallenge.market
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.yoti.android.cryptocurrencychallenge.R
+
+class MarketFragment : Fragment() {
+
+    override fun onCreateView(
+            inflater: LayoutInflater, container: ViewGroup?,
+            savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_market, container, false)
+    }
```

Comment:
Going to suggest making this even more simple by creating and registering the Adapter here with some fake data and a `TODO: Replace with real data from backend call`.

This will also let them (and us) see the expected UI before starting on the implementation. Otherwise looks good to me

Reply:
Yep good point, I have added now the adapter and some dummy data. Just one thing, is not for this Fragment as in this one we just show a detail view, not a list.

The dummy data and adapter has been added in the AssetsFragment.

Reply:
Good thing I'm not doing the coding challenge :roll_eyes: